.gitlab-ci.yml

```diff
@@ -41,6 +41,10 @@ build:
     expire_in: 2w
     paths:
       - image.tar
+  before_script:
+    # Install dependencies
+    - apt update
+    - apt install -qyy sudo
```

Comment:
It is an Alpine Linux based image, we need `apk` instead of `apt-get`

Reply:
No longer needed after ignoring rootfs_overlay_*